policy/exposures.yaml

```diff
@@ -140,3 +140,19 @@ dcrCoadd_nImage:
     python: lsst.afw.image.ImageU
     template: dcrCoadd/%(filter)s%(subfilter)dof%(numSubfilters)d/%(tract)d/%(patch)s_nImage.fits
     level: Skytile
+dcrDiff_differenceExp:
+    description: "The result of image differencing against a dcrCoadd."
+    template:      ''
+    python:        lsst.afw.image.ExposureF
+    persistable:   ExposureF
+    storage:       FitsStorage
+    level:         Ccd
+    tables:        'raw'
+dcrDiff_matchedExp:
+    description: "The result of PSF matching a dcrCoadd to the image to be differenced against."
```

Comment:
Is there a `dcrDiff_matchedExp` for each of the images being differenced, or just one of them? It's not clear from the description.

Reply:
Yes, there will be one `dcrDiff_matchedExp` for each image being differenced; I'll edit the message to make that more clear.